docs/curation-guidelines.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: default
+---
+
+# License discovery scenario guidelines
+
+We always prefer a reference to a version control system, but if you’re unable to find that, other public references are okay.
+
+1. Target component has a LICENSE or similar file embedded in a canonical location in the component (e.g., root, meta-inf folder, …) 
+    * Use the license resulting from analyzing the content of the file. If the result cannot be interpreted as an SPDX-listed license identifier, treat it as `NONE`. 
+
+2. Target component metadata has an explicit license entry indicating an SPDX (or similar) identifier 
+    * Canonicalize the identifier with an SPDX-listed license identifier and use. If the value cannot be canonicalized, treat as `NONE`. 
+
+3. Target component metadata has license info indicating a license file URL in source repo, but the link is broken or points to a volatile location. 
+    * Ignore the link specifics but use source repo URL as a candidate for source location in subsequent steps. 
```

Comment:
Repo and repository are used; would it be better to use consistent wording?

Reply:
Good catch, thank you!